model/src/gateway/payload/member_update.rs

```diff
@@ -12,7 +12,13 @@ pub struct MemberUpdate {
     /// Whether the user has yet to pass the guild's [Membership Screening]
     /// requirements.
     ///
+    /// Note: This field is still under refactoring by discord. For more info,
+    /// check this [issue] & [pull request] for more info.
```

Comment:
```suggestion
    /// check this [issue] and [pull request] for more info.
```
I think two more letters would make this nicer to read